pkg/exchange/okex/convert.go

```diff
@@ -306,3 +285,49 @@ func toGlobalOrder(okexOrder *okexapi.OrderDetails) (*types.Order, error) {
 		IsIsolated:       false,
 	}, nil
 }
+
+func toGlobalTrade(orderDetail *okexapi.OrderDetails) (*types.Trade, error) {
+	tradeID, err := strconv.ParseInt(orderDetail.LastTradeID, 10, 64)
+	if err != nil {
+		return nil, errors.Wrapf(err, "error parsing tradeId value: %s", orderDetail.LastTradeID)
+	}
+
+	orderID, err := strconv.ParseInt(orderDetail.OrderID, 10, 64)
+	if err != nil {
+		return nil, errors.Wrapf(err, "error parsing ordId value: %s", orderDetail.OrderID)
+	}
+
+	side := toGlobalSide(orderDetail.Side)
+
+	isMargin := false
+	if orderDetail.InstrumentType == string(okexapi.InstrumentTypeMARGIN) {
+		isMargin = true
+	}
+
+	isFuture := false
+	if orderDetail.InstrumentType == string(okexapi.InstrumentTypeFutures) {
+		isFuture = true
+	}
+
+	var pnl = sql.NullFloat64{Float64: orderDetail.PnL.Float64(), Valid: true}
```

Comment:
do we have pnl in every order detail?

Reply:
from Okex official site, `Last filled profit and loss, applicable to orders which have a trade and aim to close position. It always is 0 in other conditions` so it always has value, so I think I can assign this value. But I do miss something, in `/api/v5/trade/fills-history` it's "fillPnl" in response, not "pnl". I updated.

Reply:
got it